R/time.R

```diff
@@ -0,0 +1,197 @@
+#' Time Feature Generator
+#'
+#' `step_time` creates a *specification* of a recipe
+#'  step that will convert date-time data into one or more factor or
+#'  numeric variables.
```

Comment:
I think we should make it always return numeric variables - no factors (im not even sure if it returns a factor at all right now?)

Reply:
the `ampm` variables returns a factor with levels "am" and "pm"

Reply:
Will redo `ampm` variable to be 0-1 coded,